kv/error.go

```diff
@@ -86,11 +73,7 @@ func IsRetryableError(err error) bool {
 		return false
 	}
 
-	if ErrRetryable.Equal(err) ||
-		ErrLockConflict.Equal(err) ||
-		ErrConditionNotMatch.Equal(err) ||
-		// TiKV exception message will tell you if you should retry or not
-		strings.Contains(err.Error(), "try again later") {
+	if ErrRetryable.Equal(err) || ErrWriteConflict.Equal(err) {
```

Comment:
handle ErrRetryable and ErrWriteConflict here

Reply:
rename the function to `IsTxnRetryableError` is better